scrapy/templates/spiders/basic.tmpl

```diff
@@ -5,7 +5,7 @@ import scrapy
 class $classname(scrapy.Spider):
     name = '$name'
     allowed_domains = ['$domain']
-    start_urls = ['$start_url/']
+    start_urls = ['http://$domain/']
```

Comment:
if a user wants to crawl web-page with `https://` ?
And `hard coding` value is never a good idea.

Reply:
I think you are commenting on a code that's not changed in this PR.